### PR TITLE
Implement reduceMin

### PR DESCRIPTION
**Pull request by qti3e**
and add tensor.size
I will need these APIs to implement NumPy's `array2string` (`tensor.toString()`)

https://github.com/numpy/numpy/blob/3629e3986333ecdf394924e53265e0d1061e928d/numpy/core/arrayprint.py#L1104-L1106

### CODE REVIEW

File: src/tensor.ts [hunk 1]
```diff
@@ -169,6 +169,20 @@ export class Tensor implements types.Storage {
     return this.storage.shape;
   }
 
+  /** Returns size of the tensor
```
Comment (ry) [inline src/tensor.ts]: Returns the total number of elements

File: src/tensor.ts [hunk 2]
```diff
@@ -169,6 +169,20 @@ export class Tensor implements types.Storage {
...
+   *
+   *    import { range } from "propel";
+   *    range(200).reshape([100, 2]).size
+   */
+  get size(): number {
+    const shape = this.storage.shape;
```
Comment (ry) [inline src/tensor.ts]: return this.shape.reduce((a, b) => a * b)

File: src/tensor.ts (hunk 1 above)
Comment (qti3e) [reply src/tensor.ts]: done

File: src/tensor.ts (hunk 2 above)
Comment (qti3e) [reply src/tensor.ts]: done

**Pull request merged**